gym/lib/gym/generators/package_command_generator_legacy.rb

```diff
@@ -11,7 +11,7 @@ module Gym
   class PackageCommandGeneratorLegacy
     class << self
       def generate
-        parts = ["/usr/bin/xcrun #{XcodebuildFixes.patch_package_application} -v"]
+        parts = ["/usr/bin/xcrun '#{XcodebuildFixes.patch_package_application}' -v"]
```

Comment:
Nice catch, however the correct fix should use shellescape without any `'`. Make sure to also require the library before calling the method:

```
xcrun XcodebuildFixes.patch_package_application.shellescape -v
```